src/Forms/FormHandler.php

```diff
@@ -1510,22 +1511,26 @@ public function perform_add(string &$form_action): void
                     $row["validation_err"] = $result;
                     if ($result !== true) {
                         $this->all_fields_valid = false;
-                        $form_action = "ask-add";
+                        $form_action = "ask-edit";
```

Comment:
Guessing this was a copy/paste error. Tried to update but didn't have permissions to edit the PR. I'll fix after merge.